packages/store/src/execution/dispatchOutsideZoneNgxsExecutionStrategy.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  NgZone,
+  PLATFORM_ID,
+  Injectable,
+  ɵNoopNgZone as NoopNgZone,
+  Inject
+} from '@angular/core';
+import { isPlatformServer } from '@angular/common';
+
+import { NgxsExecutionStrategy } from '../symbols';
+import { ConfigValidator } from '../internal/config-validator';
+
+@Injectable()
+export class DispatchOutsideZoneNgxsExecutionStrategy implements NgxsExecutionStrategy {
+  constructor(private _ngZone: NgZone, @Inject(PLATFORM_ID) private _platformId: Object) {
+    this.verifyZoneIsNotNooped(this._ngZone);
+  }
+
+  enter<T>(func: (...args: any[]) => T): T {
```

Comment:
enter<T>(callback: Callback<T>)

and others

Reply:
A callback would be something that would happen on completion of some work IMO.
Here the function is describing the work to do, not something to call aftet the work is complete.
There is a distinction in my mind... but maybe it is just me?